packages/shared/lib/contexts/onboarding/helpers/setImportFile.ts

```diff
@@ -0,0 +1,6 @@
+import { importFile, importFilePath } from '../stores'
+
+export function setImportFile(file: ArrayBuffer, filePath: string): void {
+    importFile.set(<Buffer>file)
```

Comment:
I think the argument should be `ArrayBuffer | string` and no type casting should happen on l.4. Related comment in FileImportView